### ISSUE TITLE
VSIX

### ISSUE THREAD

**Issue by minkir014** [tokoro10g/matlab-langserver#1]
Your work is great but it would easier for users to make a vsix version which can run in vscode instead of everyone doing the same thing and with every commit they must update content of you built it you'll have more users.

**Comment by tokoro10g** [comment]
Thanks for the positive and constructive comment! I agree with you, and I think it would be provided as a combo of this language server implementation and the corresponding LSP client.

This project is still in an experiment, and currently I don't plan to provide any release builds.
Also, redistribution of engine.jar file from MATLAB is prohibited, so I'll have to find an appropriate way to keep the references to MATLAB Engine methods without contaminations.

**Comment by minkir014** [comment]
You can make a experimental vsix. You can push it in GitHub and not on the marketplace as an experiment.

**Comment by minkir014** [comment]
Could you at least provide a good readme that shows the abilities of this extension? And why is redistribution of engine.jar file from MATLAB prohibited?

**Comment by tokoro10g** [comment]
Yes, I will add README! Please be patient because I am busy these days.

>And why is redistribution of engine.jar file from MATLAB prohibited?

It is due to the license agreement of MATLAB.

Since it is becoming off-topic, I suggest you to open new issues if you have further requests on this project.
